sdk/identity/ci.yml

```diff
@@ -73,12 +73,25 @@ extends:
     LiveTestStages:
       - template: /eng/pipelines/templates/stages/archetype-sdk-tests-isolated.yml
         parameters:
+          PreTestRunSteps:
```

Comment:
This step shouldn't be needed if you set PersistOidcToken to true

Reply:
The token lifetime is only 10 minutes and the value in env var doesn't auto renew.

Reply:
@g2vinay if you add a re-login to the pre step I believe it should work, like I did here: https://github.com/Azure/azure-sdk-for-go/pull/23722/files#diff-3af3a621938c060cdd62334693f70822ab9065703da3ad7cb75d6fcfb7b536beR51

Reply:
Oh, I see now you are doing that below. Is it still an issue?